src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerCoordinator.swift

```diff
@@ -385,7 +385,8 @@ class QRScannerCoordinator {
 					}
 
 					self.ticketValidationCoordinator = TicketValidationCoordinator(
-						parentViewController: parentViewController
+						parentViewController: parentViewController,
+						healthCertificateService: self.healthCertificateService
```

Comment:
please have a short looking, in line 380 is no weak self set for the closure, this might leak. Could you please double check if one might be missing here?